cmd/commands/service/command.go

```diff
@@ -90,24 +119,14 @@ func runServices(ctx *cli.Context, di *cmd.Dependencies, licenseCommandName stri
 	// 1 for each of the services
 	errorChannel := make(chan error, 2+len(serviceTypes))
 
-	nodeOptions := cmd.ParseFlagsNode(ctx)
-	if err := di.Bootstrap(nodeOptions); err != nil {
-		return err
-	}
-	if err := di.BootstrapServices(nodeOptions); err != nil {
-		return err
-	}
-
 	go func() { errorChannel <- di.Node.Wait() }()
 
 	for _, serviceType := range serviceTypes {
 		options, err := parseFlagsByServiceType(ctx, serviceType)
 		if err != nil {
 			return err
 		}
-		go func(serviceType string) {
-			errorChannel <- di.ServiceRunner.StartServiceByType(serviceType, options)
-		}(serviceType)
+		go func() { errorChannel <- di.ServiceManager.Start(providerID, serviceType, options) }()
```

Comment:
It's better to pass options variable to the func, since it could be overwritten on the next iteration, even before first goroutine start. 
> loop variable serviceType captured by func literal (govet)

Reply:
Fixed

Reply:
Remaked